packages/calling/calling/src/CallingClient/line/index.ts

```diff
@@ -0,0 +1,463 @@
+      );
+
+      try {
+        const temp = <WebexRequestPayload>await this.#webex.request({
+          uri: `${
+            this.#webex.internal.services._serviceUrls.mobius
+          }${URL_ENDPOINT}?regionCode=${clientRegion}&countryCode=${countryCode}`,
+          method: HTTP_METHODS.GET,
+          headers: {
+            [CISCO_DEVICE_URL]: this.#webex.internal.device.url,
+            [SPARK_USER_AGENT]: CALLING_USER_AGENT,
+          },
+          service: ALLOWED_SERVICES.MOBIUS,
+        });
+
+        log.log('Mobius Server found for the region', '' as LogContext);
+        const mobiusServers = temp.body as MobiusServers;
+
+        /* update arrays of Mobius Uris. */
+        const mobiusUris = filterMobiusUris(
+          mobiusServers,
+          this.#webex.internal.services._serviceUrls.mobius
+        );
+        this.#primaryMobiusUris = mobiusUris.primary;
+        this.#backupMobiusUris = mobiusUris.backup;
+        log.info(
+          `Final list of Mobius Servers, primary: ${mobiusUris.primary} and backup: ${mobiusUris.backup}`,
+          '' as LogContext
+        );
+      } catch (err: unknown) {
+        handleRegistrationErrors(
+          err as WebexRequestPayload,
+          (clientError) => {
+            this.#registration.sendMetric(
+              METRIC_EVENT.REGISTRATION_ERROR,
+              REG_ACTION.REGISTER,
+              METRIC_TYPE.BEHAVIORAL,
+              clientError
+            );
+            this.emit(LINE_EVENTS.ERROR, clientError);
+          },
+          {method: GET_MOBIUS_SERVERS_UTIL, file: LINE_FILE}
+        );
+
+        useDefault = true;
+      }
+    } else {
+      /* Setting this to true because region info is possibly undefined */
+      useDefault = true;
+    }
+
+    /* Use a default URL if Mobius discovery fails either because of region info failure
+     * or because the discovered Mobius couldn't be reached
+     */
+
+    if (useDefault) {
+      log.warn('Error in finding Mobius Servers. Will use the default URL.', '' as LogContext);
+      this.#primaryMobiusUris = [
+        `${this.#webex.internal.services._serviceUrls.mobius}${URL_ENDPOINT}`,
+      ];
+    }
+  }
+
+  /**
+   * Wrapper to for device registration.
+   *
+   * @param retry - Retry in case of errors.
+   */
+  public async register() {
+    await this.#mutex.runExclusive(async () => {
+      this.#registration.setStatus(MobiusStatus.DEFAULT);
+      this.emit(LINE_EVENTS.CONNECTING);
+
+      /* Don't do a discovery if we already have the servers list */
+      if (this.#primaryMobiusUris.length === 0) {
+        await this.getMobiusServers();
+      }
+
+      this.#registration.setMobiusServers(this.#primaryMobiusUris, this.#backupMobiusUris);
+      await this.#registration.triggerRegistration();
+    });
+  }
+
+  /**
+   * Wrapper to for device  deregister.
+   */
+  public async deregister() {
+    this.#registration.deregister();
+  }
+
+  // will be done as part of call related jira
```

Comment:
We can remove these changes for now. Not required

Reply:
Removed